power_grid_model_c/power_grid_model/include/power_grid_model/main_core/state_queries.hpp

```diff
@@ -41,7 +41,7 @@ constexpr Idx get_component_group_idx(MainModelState<ComponentContainer> const&
 
 template <typename ComponentType, class ComponentContainer>
     requires model_component_state_c<MainModelState, ComponentContainer, ComponentType>
-inline Idx get_component_sequence(MainModelState<ComponentContainer> const& state, auto const& id_or_index) {
+inline Idx get_component_sequence_idx(MainModelState<ComponentContainer> const& state, auto const& id_or_index) {
```

Comment:
For this PR I'll leave this one as is, but I will add it to the discussion for the major overhaul, because if we dive deep into this, we may as well unify it accross the whole codebase.

Reply:
https://github.com/PowerGridModel/power-grid-model/issues/839

Reply:
> #839

Resolving due to this.